test/unit/specs/data/json-update-builder.js

```diff
@@ -0,0 +1,154 @@
+import { describe, it } from 'mocha'
+
+import Backendless from '../../helpers/sandbox'
+import { expect } from 'chai'
+
+describe('<Data> JSON Update Builder', function() {
+  let jsonUpdateBuilder
+
+  beforeEach(() => {
+    jsonUpdateBuilder = Backendless.Data.JSONUpdateBuilder
+  })
+
+  it('check default values', async () => {
+    const jsonUpdateBuilderInstance = new jsonUpdateBuilder()
```

Comment:
and then here it will look like: `const jsonUpdateBuilder = new JSONUpdateBuilder()`

Reply:
+